src/Files.App/Helpers/NativeWinApiHelper.cs

```diff
@@ -247,17 +249,13 @@ public static bool IsHasThreadAccessPropertyPresent
 		{
 			get
 			{
-				if (isHasThreadAccessPropertyPresent is null)
-				{
-					isHasThreadAccessPropertyPresent =
-						Windows.Foundation.Metadata.ApiInformation.IsPropertyPresent(typeof(Windows.System.DispatcherQueue).FullName, "HasThreadAccess");
-				}
+				isHasThreadAccessPropertyPresent ??= ApiInformation.IsPropertyPresent(typeof(DispatcherQueue).FullName,"HasThreadAccess");
 				return isHasThreadAccessPropertyPresent ?? false;
 			}
 		}
 
 		// https://www.travelneil.com/wndproc-in-uwp.html
-		[ComImport, Guid("45D64A29-A63E-4CB6-B498-5781D298CB4F")]
+		[ComImport, System.Runtime.InteropServices.Guid("45D64A29-A63E-4CB6-B498-5781D298CB4F")]
```

Comment:
Why add namespace here?

Reply:
Line 250 to 254 had `Windows.Foundation.Metadata.ApiInformation` and `Windows.System` namespaces. I made them into `using` statements. 

It creates an ambiguity on the `Guid` type that could be either from `System.Runtime.InteropServices` or `Windows.Foundation.Metadata`. So it's here for disambiguation.

If we prefer having on `Guid` on l.258, I can remove the `using` statements at put it as it was previously.